iocore/hostdb/HostDB.cc

```diff
@@ -2083,7 +2083,7 @@ void
 HostDBRecord::free()
 {
   if (_iobuffer_index > 0) {
-    Debug("hostdb", "freeing %d bytes at [%p]", (1 << (7 + _iobuffer_index)), this);
```

Comment:
Did you mean to remove this debug output?

Reply:
oh yeah, this causes a crash on exit so I commented it out